melos.yaml

```diff
@@ -28,3 +30,20 @@ scripts:
 
   format:
     exec: dart format lib test -l 80 --set-exit-if-changed
+
+  update-version:
+    run: |
+      # Loop through the packages directory
+      for d in packages/*/ ; do
+        # Get version number
+        version=$(awk '/version:/ { 
+          pos=match($2, /[0-9]+\.[0-9]+\.[0-9]+/);
+          print substr($2, pos, RLENGTH); }' $d/pubspec.yaml)
+        # Update version.dart file
+        echo "const version = '$version';" > $d/lib/src/version.dart
+      done
+      # Remove the version file for yes_another_json_isolate
+      rm packages/yet_another_json_isolate/lib/src/version.dart
+      # Stage the version.dart file change
+      git add .
```

Comment:
In case you have some other changes locally, I don't think it's a good idea to add all files. What about `git add **/version.dart` instead.

Reply:
Updated it!